app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
 
+  validates :name, presence: { message: 'Name must not be blank' }
```

Comment:
- [ ] It would be better if we ensure that the field name (name) should be present and not empty. And the best way to do that is `presence: true` instead of `presence: { message: 'Name must not be blank' }` as we have currently

Reply:
Thank you for the review and for the help in fixing that!!